src/seedu/addressbook/commands/FindCommand.java

```diff
@@ -1,13 +1,11 @@
 package seedu.addressbook.commands;
 
-import java.util.ArrayList;
-import java.util.Collections;
-import java.util.HashSet;
-import java.util.List;
-import java.util.Set;
+import java.util.*;
```

Comment:
Collapsing imports is against our coding standard. Make sure you set IntelliJ to not do this: Preferences > Editor > Code Style > Java > Imports tab set Class count to use import with '*'